pkgs/development/libraries/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  nix-update-script,
+
+  # base build deps
+  meson,
+  pkg-config,
+  ninja,
+
+  # docs build deps
+  python3,
+  doxygen,
+  graphviz,
+
+  # GI build deps
+  gobject-introspection,
+
+  # runtime deps
+  glib,
+  systemd,
+  lua5_4,
+  pipewire,
+
+  # options
+  enableDocs ? true,
+  enableGI ? stdenv.hostPlatform == stdenv.buildPlatform
+}:
+let
+  mesonEnable = b: if b then "enabled" else "disabled";
+in stdenv.mkDerivation rec {
+  pname = "wireplumber";
+  version = "0.4.6";
+
+  outputs = [ "out" "dev" ] ++ lib.optional enableDocs "doc";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "pipewire";
+    repo = "wireplumber";
+    rev = version;
+    sha256 = "sha256-y+Gj9EZn67W3U81zXgp+6JAFxZSZTwwT0TB3Kueb/Tw=";
+  };
+
+  mesonFlags = [
```

Comment:
Same attribute ordering nitpick with `mesonFlags`. You can use https://github.com/jtojnar/nixpkgs-hammering to check for it automatically.

Reply:
nixpkgs-hammering passes now.